api/apps/api/src/modules/scenarios/marxan-run/assets.service.ts

```diff
@@ -0,0 +1,45 @@
+import { Assets } from '@marxan/scenario-run-queue';
+import { FactoryProvider, Inject, Injectable } from '@nestjs/common';
+import { AppConfig } from '@marxan-api/utils/config.utils';
+import { IoSettings, ioSettingsToken } from './io-settings';
+
+export const apiUrlToken = Symbol('api url token');
+export const apiUrlProvider: FactoryProvider<string> = {
+  provide: apiUrlToken,
+  useFactory: () => AppConfig.get<string>('api.url'),
+};
```

Comment:
don't we usually keep those in standalone file? (like for api/apps/api/src/modules/scenarios/marxan-run/io-settings.ts below)

Reply:
io-settings previously also was in one file with its user. I tend to make it a separate files when it has more than one user.